doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2719,6 +2744,23 @@ Peripheral HIDS keyboard sample cannot be used with nRF Bluetooth LE Controller
   The library uses the MPSL Clock driver, which does not provide an API for asynchronous clock operation.
   NFC requires this API to work correctly.
 
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0 v2-7-0
```

Comment:
Please remove the v3-0-0 tag since the fix is merged as part of the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/21844

```suggestion
.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0 v2-7-0
```